openreview/conference/templates/tabsConferenceDecisionsWebfield.js

```diff
@@ -68,61 +116,86 @@ function renderContent(notes, withdrawnNotes, decisionsNotes) {
       papersByDecision[d.content.decision].push(notesDict[d.forum]);
     }
   });
-
   for (var decision in DECISION_HEADING_MAP) {
     papersByDecision[getElementId(decision)] = _.sortBy(papersByDecision[decision], function(o) { return o.id; });
   }
 
-  var paperDisplayOptions = {
-    pdfLink: true,
-    replyCount: true,
-    showContents: true
-  };
-
-  var activeTab = 0;
-  var loadingContent = Handlebars.templates.spinner({ extraClasses: 'spinner-inline' });
-  var sections = [];
-
-  for (var decision in DECISION_HEADING_MAP) {
-    sections.push({
-      heading: DECISION_HEADING_MAP[decision],
-      id: getElementId(decision),
-      content: loadingContent
-    });
-  }
-
-  sections[activeTab].active = true;
-  sections[activeTab].content = null;
-
-  $('#notes .tabs-container').remove();
-
-  Webfield.ui.tabPanel(sections, {
-    container: '#notes',
-    hidden: true
-  });
-
   $('#group-container').on('shown.bs.tab', 'ul.nav-tabs li a', function(e) {
+    var activeTab = 0;
     activeTab = $(e.target).data('tabIndex');
     var containerId = sections[activeTab].id;
-
     setTimeout(function() {
-      Webfield.ui.searchResults(
-        papersByDecision[containerId],
-        _.assign({}, paperDisplayOptions, {showTags: false, container: '#' + containerId})
-      );
+      if (activeTab < Object.keys(DECISION_HEADING_MAP).length) {
+        Webfield.ui.searchResults(
+          papersByDecision[containerId],
+          _.assign({}, paperDisplayOptions, {showTags: false, container: '#' + containerId})
+        );
+      }
     }, 100);
   });
 
+  var withdrawnNotesCount = withdrawnNotes.count || 0;
```

Comment:
adding the notes to the map doesn't work?

Reply:
I think we should keep these 2 tabs separate from the map as the map i.e. DECISION_HEADING_MAP contains notes for which decisions are available. I could still merge all results in one map if you feel strongly about it.